### PR TITLE
Fix devel branch after YARP 3.0.0 release

### PR DESCRIPTION
**Pull request by Nicogene**
This PR fixes devel branch after the release of YARP 3.0.0
cc @traversaro

### CODE REVIEW

File: src/libraries/icubmod/embObjMotionControl/eomcParser.h [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
 #include <yarp/dev/PidEnums.h>
 #include <yarp/dev/ControlBoardInterfacesImpl.h>
-#include <yarp/dev/ControlBoardInterfacesImpl.inl>
+#include <yarp/dev/ControlBoardInterfacesImpl.h>
```
Comment (traversaro) [inline src/libraries/icubmod/embObjMotionControl/eomcParser.h]: Duplicated header include.

File: src/libraries/skinDynLib/include/iCub/skinDynLib/skinContact.h [hunk 2]
```diff
@@ -269,7 +269,7 @@ class skinContact : public dynContact
     * 16-18: surface normal direction; 
     * 19: number of active taxels;
     * 20-?: list of the id's of the activated taxels; 
-    * ?: pressure;
+    * ?: pressure;const
```
Comment (traversaro) [inline src/libraries/skinDynLib/include/iCub/skinDynLib/skinContact.h]: leftover

**Comment by traversaro** [comment]
I just realized that probably there are a lot of devices that are broken, and that are not fixed by this PR. 
See http://wiki.icub.org/wiki/Compilation_on_the_pc104#List_of_modules for a list (note: to compile them you need to have a recent icub-firmware-shared installation).

**Comment by Nicogene** [comment]
I have the latest `icub-firmware-shared` on my pc and these are the devices that I can compile and that compiles:

image

**Pull request merged**